Mathlib/Algebra/Group/Hom/End.lean

```diff
@@ -4,9 +4,9 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Patrick Massot, Kevin Buzzard, Kim Morrison, Johan Commelin, Chris Hughes,
   Johannes Hölzl, Yury Kudryashov
 -/
-import Mathlib.Algebra.Group.Commute.Defs
 import Mathlib.Algebra.Group.Hom.Instances
-import Mathlib.Algebra.Ring.Basic
+import Mathlib.Algebra.Ring.Defs
+import Mathlib.Tactic.MinImports
```

Comment:
```suggestion
```

Reply:
Why? I am adding this import wherever I could not use `#min_imports`, because one should be able to use `#min_imports` everywhere.

I am also happy to add it to `Mathlib.Init` FWIW

Reply:
If we want this available everywhere, then it should be in `Init` (and should be in a standalone PR)

Reply:
Moved to #[24722](https://github.com/leanprover-community/mathlib4/pull/24722)